packages/client/src/components/MovieComponent.js

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { Card, CardMedia, Typography, Rating, Box } from '@mui/material';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+
+const BasicRating = () => {
+  return (
+    <Rating
+      name="read-only"
+      value={4}
```

Comment:
I understand this is a template and this is value(s) are hardcoded, right?

Reply:
Yes, that's right. It could be replaced with real data later.